sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala

```diff
@@ -68,6 +71,39 @@ class SidechainNodeViewHolder(sidechainSettings: SidechainSettings,
       secretModify(ls.secret)
   }
 
+  import Certifier.ReceivableMessages._
+
+  protected def sendBakwardTransferCertificate: Unit = {
+    certifier match {
+      case Some(c) => {
+        val withdrawalEpochInfo = minimalState.getWithdrawalEpochInfo
+        val startEpochHeight = (withdrawalEpochInfo.lastEpochIndex -
+          sidechainSettings.genesisData.mcBlockHeight) %
+          sidechainSettings.genesisData.withdrawalEpochLength + 1
+        if (withdrawalEpochInfo.epoch > 1 &&
```

Comment:
` > 0`. We start epoch calculation from 0, so in epoch 1 we already expect Certificate.